examples/mobile/HomeApp/index.html

```diff
@@ -45,6 +45,43 @@
 				</a>
 			</div>
 
+			<div class="ui-content-subheader">
+				Hotel Restaurant
+			</div>
+			<div class="ui-content-area">
+				<div class="app-hotel-breakfast">
+					<div class="app-hotel-breakfast-image"></div>
+					<ul class="ui-listview app-hotel-breakfast-description">
+						<li class="ui-li-divider">
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									English Breakfast
+								</span>
+								<span class="ui-li-text-sub">
+									Delicious breakfast: fried egg, baked bacon, beans, sausages, tomato, bread, hot drink.
+								</span>
+							</div>
+							<div class="app-hotel-order">
+								<a href="#" class="ui-btn ui-btn-contained-colored" data-inline="true">Order</a>
```

Comment:
can we point to breakfast-menu.html
so we will be compatible with https://github.com/Samsung/TAU/pull/1346/